scormxblock/scormxblock.py

```diff
@@ -506,6 +510,22 @@ def _publish_progress(self, completion):
         """
         self.runtime.publish(self, 'completion', {'completion': completion})
 
+    def calculate_progress_measure(self, scorm_data):
+        """
+        Returns the averaged progress_measure of all scos in the current scorm content
+        :return: progress_measure if found, else 0
+        """
+        progress_sum = 0
+        scos = scorm_data.get('scos', [])
+        for sco in scos.values():
```

Comment:
This will give values error if there are not scos and L519 returns an empty array.